src/openvic-simulation/dataloader/Dataloader.cpp

```diff
@@ -10,6 +10,7 @@
 #include "openvic-simulation/DefinitionManager.hpp"
 #include "openvic-simulation/utility/Logger.hpp"
 #include "openvic-simulation/utility/StringUtils.hpp"
+#include "GameRulesManager.hpp"
```

Comment:
```suggestion
#include "openvic-simulation/GameRulesManager.hpp"
```
This include isn't really needed anyway, I suspect clangd added it automatically?